ocaml/xapi-idl/xen/xenops_interface.ml

```diff
@@ -465,11 +465,11 @@ module Host = struct
     ; model: string
     ; stepping: string
     ; flags: string
-    ; features: int64 array
-    ; features_pv: int64 array
-    ; features_hvm: int64 array
-    ; features_pv_host: int64 array
-    ; features_hvm_host: int64 array
+    ; features: string
```

Comment:
This thing is that this needs to be a string in order for xapi to store in the database, and it needs to be serialisable for ocaml-rpc as well. I don't think it can be abstract in the interface.

Reply:
this got changed to 'host_cpu_policy'.